.circleci/config.yml

```diff
@@ -5,9 +5,10 @@ executors:
   android-executor:
     docker:
       - image: circleci/android:api-28
+    resource_class: medium
```

Comment:
once we upgrade the circleCI plan, this should probably be medium+ so we can comfortably cover the memory used for the JVM

Reply:
yes, I didn't request @MiguelCarranza to upgrade because I figured out the other issue, but it's not giving OOM anymore